python/ray/tune/examples/skopt_example.py

```diff
@@ -63,9 +64,10 @@ def easy_objective(config, reporter):
     algo = SkOptSearch(
         optimizer, ["width", "height"],
         max_concurrent=4,
-        reward_attr="neg_mean_loss",
+        metric="neg_mean_loss",
+        mode="min",
         points_to_evaluate=previously_run_params)
-    scheduler = AsyncHyperBandScheduler(reward_attr="neg_mean_loss")
+    scheduler = AsyncHyperBandScheduler(metric="neg_mean_loss", mode="max")
```

Comment:
Can you? do metric="mean_loss" and mode="min" for all search/schedulers here too?